### PR TITLE
Drop travis tests

### PR DESCRIPTION
**Pull request by omeryahud**
**What this PR does / why we need it**:
This PR retired Travis from running our tests after we have successfully moved on to OCP CI.
From now on, travis would only generate release manifests when we create new releases.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,57 +13,23 @@ services:
...
-
   - stage: Build and Deploy
     name: Build and Deploy to GitHub
     if: tag IS present
     before_script: skip
     script: make release "IMAGE_TAG=$TRAVIS_TAG"
     deploy:
       provider: releases
-      api_key: "$GITHUB_TOKEN"
+      api_key: "$GITHUB_RELEASE_TOKEN"
```
Comment (ksimon1) [inline .travis.yml:23]: do you set correct variable in travis?

Comment (omeryahud) [reply .travis.yml:23]: Yes, https://travis-ci.com/github/kubevirt/kubevirt-ssp-operator/settings

**Comment by ksimon1** [comment]
@omeryahud 
Is this PR ready to be merged?

**Pull request closed without merging**